api/src/main/java/jakarta/servlet/http/HttpServletResponse.java

```diff
@@ -145,9 +145,57 @@ public interface HttpServletResponse extends ServletResponse {
     public void sendError(int sc) throws IOException;
 
     /**
-     * Sends a temporary redirect response to the client using the specified redirect location URL and clears the buffer.
-     * The buffer will be replaced with the data set by this method. Calling this method sets the status code to
+     * Sends a temporary redirect response to the client using the specified redirect location URL using the status code
+     * {@link #SC_FOUND} 302 (Found) and clears the buffer. The buffer will be replaced with the data set by this method.
+     *
+     * @param location the redirect location URL
```

Comment:
Can we mention that the location can be absolute or relative here in the javadoc (and not in the intro to the full sendRedirect method). 

We should probably also list he IllegalArgumentException that the spec says should be used if a relative URI can not be converted.

Reply:
Agreed. I'll update the PR.